logo-white.svg

```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- SPDX-License-Identifier: CC0-1.0 -->
 <svg
    xmlns:dc="http://purl.org/dc/elements/1.1/"
-   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:cc="https://creativecommons.org/publicdomain/zero/1.0/"
```

Comment:
XMLNS:* (XML Namespace decleration) is like a PREFIX decleration in SPARQL and does not effect the license. `http://creativecommons.org/ns#` is a vocabulary for various license related concepts, in this file the only used concept is: `http://creativecommons.org/ns#Work`. This change changes it to `https://creativecommons.org/publicdomain/zero/1.0/Work`.